app/views/publishers/keys/new.html.slim

```diff
@@ -0,0 +1,6 @@
+.dashboard
+  .row
+    .col.mb-4
+      .rounded.bg-white.p-4
+        .d-flex.align-items-center.justify-content-between.mb-3
+          = render "new"
```

Comment:
This is slightly confusing. Is it possible to change this file or this rendering?

Reply:
Nevermind.